todo/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["name"]
+
+    def __str__(self):
+        return f"{self.name}"
+
+
+class Task(models.Model):
+    content = models.CharField(blank=True, max_length=15)
+    created_at = models.DateTimeField(auto_now_add=True)
+    deadline = models.DateTimeField()
+    is_done = models.BooleanField(default=False)
+    tags = models.ManyToManyField(to=Tag, related_name="tasks")
+
+    class Meta:
+        ordering = ["created_at", "is_done"]
```

Comment:
Tasks should be ordered from not done to done and from newest to oldest. In that case we are ordering from oldest to newest and after that based on `is_done` status.

```suggestion
    class Meta:
        ordering = ["is_done", "-created_at"]
```